src/elisp/treemacs-interface.el

```diff
@@ -272,6 +272,11 @@ Stay in current window with a prefix argument ARG."
    :save-window arg
    :no-match-explanation "Node is neither a file, a directory or a tag - nothing to do here."))
 
+(defun treemacs-visit-node-default (&optional arg)
+  "Run `treemacs-default-visit-action' for the current button.
+A potential prefix ARG is passed on to the executed action, if possible."
+  (funcall treemacs-default-visit-action arg))
```

Comment:
Should be interactive, or defined elsewhere. The -interface module is for interactive, but not autoloaded functions.

Reply:
I'll make it interactive - I can imagine people wanting to bind this to some key.